### DISCUSSION TITLE
2024/04/23/index

### DISCUSSION THREAD

**Discussion by giscus** [embedded-kiddie/utterances-comments#6] [Announcements]
# 2024/04/23/index

GitHub Pages 上で使えるコメントフォームの UI や使い心地を確認するため、５つのホスティングサービスを実際に試してみました。全て フリーミアム もしくは無料で設置できます。

http://localhost:4000/2024/04/23/index.html

**Comment by embedded-kiddie** [comment]
utterances もそうですが、emoji が単独行でないとうまく表示できませんね。:grinning:
:stuck_out_tongue_winking_eye: :100:
